### ISSUE TITLE
fatal: You don't exist. Go away!

### ISSUE THREAD

**Issue by Gwan-Wang** [meanjs/generator-meanjs#191]
You're using the official MEAN.JS generator.
? What mean.js version would you like to generate? master
master
? In which folder would you like the project to be generated? This can be changed later. mean
Cloning the MEAN repo.......
Error: Command failed: /bin/sh -c git clone --branch master https://github.com/meanjs/mean.git mean
fatal: You don't exist. Go away!

why? i can solve this problem, i need your help,thank

**Comment by ilanbiala** [comment]
@Gwan-Wang are you on Windows?

**Comment by Gwan-Wang** [comment]
@ilanbiala no, my os is centOS (Linux iZ28h29kk0fZ 2.6.32-573.12.1.el6.x86_64 #1 SMP Tue Dec 15 21:19:08 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux);

**Comment by ilanbiala** [comment]
We haven't tested against centOS, but my initial guess is that git isn't installed.

**Comment by Gwan-Wang** [comment]
@ilanbiala maybe you can test in centOS6.5 mini, and i have install git(1.7.1), if i find answer, i will write here

**Comment by ilanbiala** [comment]
@Gwan-Wang I'm not sure how we can test against other OSes other than Mac on Travis. @codydaig @lirantal any ideas?

**Comment by codydaig** [comment]
@ilanbiala 
https://github.com/meanjs/mean/pull/1184
https://travis-ci.org/meanjs/mean/builds/106835692
I'll also add these for the generator repo if everything checks out on the main repo.

**Comment by ilanbiala** [comment]
:+1:, though I don't think Travis does Windows, AFAIK we need Appveyor.

**Comment by codydaig** [comment]
@ilanbiala #193 would have Travis test windows.
